src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -988,8 +988,11 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
 
     // check if the bank has enough memory to allocate
     // m_size is in KB so convert block_size (bytes) to KB for comparison
-    if (mem.m_size < (block_size/1024))
+    if (mem.m_size < (block_size/1024)) {
+      logger(_ptTest, "Details", boost::str(boost::format(
+	"The bank does not have enough memory to allocate. Use lower '%s' value. \n") % "block-size"));
```

Comment:
By reviewing the log message, it is better to inform the bank index also while printing this message.